Fix for 1.4

Nucleoprotein opened the pull request:
Quick fix for crash at world generation because of changed APIs

Nucleoprotein committed:
Fix for 1.4

neitsa commented:
Damn that's awesome! Let me check on my side if everything works as excepted (there's no reason it shouldn't work, but I prefer to test just to be sure there's no showstopper). Once tested, I'll do a build and push the artifacts on Steam. Week's busy at work, so expected ETA is next Saturday.

Oh, and thank you for inverting that dang OR / AND filters. Dunno what I was thinking at that time...

Can I credit you w/ "Nucleoprotein" or would you like to use another nickname? (here on the changelog and on steam).

Once again thanks a lot for your work!

Nucleoprotein commented:
Thanks,
I have check that you are not very active on github/steam so i fixed it for myself but if i already fixed it why not to share 🙂
Nucleoprotein for credit is ok 🙂
